src/components/ebay-drawer-dialog/index.marko

```diff
@@ -10,6 +10,7 @@ $ var handleLabel = (!state.expanded ?
     on-scroll('handleScroll')
     on-open("emit", "open")
     on-close("emit", "close")
+    on-toggled("emit", "toggled")
```

Comment:
I think this may be a remnant from a previous attempt, but we should bring it back to the base component so it can be used for tracking and such. It also may be renamed to something like `on-close-click`, but that is up to you.